src/Autocomplete/src/Maker/skeletons/AutocompleteField.tpl.php

```diff
@@ -23,17 +22,12 @@ public function configureOptions(OptionsResolver $resolver)
             'placeholder' => 'Choose a <?php echo $variables->entityClassDetails->getShortName(); ?>',
```

Comment:
Could you add a `: void` return type on configurationOptions(..) while you're on this ? 👼 

Thank you :)

Reply:
done 👍